### ISSUE TITLE
Opening bracket for if-else is added to a commented line (output won't compile)

### ISSUE THREAD

**Issue by ming-zhang** [pinterest/ktlint#944]
## Source Code
```kt
val x = 5
val output = if (x > 0) 
// Greater than zero
    "Positive"
else
// Not greater than zero
    "Not positive"
```

## Expected Behavior
`ktlint --experimental --format` should yield:

```kt
val x = 5
val output = if (x > 0) {
    // Greater than zero
    "Positive"
} else {
    // Not greater than zero
    "Not positive"
}
```

## Observed Behavior
Generates code that doesn't compile:

```kt
val x = 5
val output = if (x > 0)
// Greater than zero {
    "Positive"
} else
// Not greater than zero {
    "Not positive"
}
```

## Your Environment

* Version of ktlint used: 0.39.0
